src/dropdown/dropdown.scss

```diff
@@ -217,7 +220,10 @@ $height-small: 30px !default;
   &.is-open {
     background: $background-open;
     border-color: $border-color-open;
-    color: $text-color-open;
+
+    &:hover > .Select-control .Select-value .Select-value-label {
+      color: $text-color-open;
```

Comment:
again, this is called `$text-color-open` but you're using only when hovering (as if it was `text-color-open-hover`)